src/App.jsx

```diff
@@ -14,70 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const stateOfGood = (good) => {
```

Comment:
same here)
let's fix naming. There is a common used event handlers namin convention. Check [this](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) out

```suggestion
  const handleGoodClick = (good) => {
```